src/Fixer/BlankLineAfterControlStructureFixer.php

```
@@ -0,0 +1,232 @@
+    {
+        return new FixerDefinition(
+            'A control structure is separated from the statement after it by a blank line.',
+            [
+                new CodeSample(
+                    "<?php\nif (\$a) {\n    b();\n}\nc();\n",
+                ),
+            ],
+        );
+    }
+
+    public function fix(SplFileInfo $file, Tokens $tokens): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$file'. (undefined)

(UnusedFormalParameter)